curriculum/challenges/english/10-coding-interview-prep/rosetta-code/happy-numbers.md

```diff
@@ -84,7 +84,7 @@ assert(happy(28));
 assert(happy(31));
 ```
 
-`happy(32)` should return true:.
+`happy(32)` should return true.
```

Comment:
Since we are currently attempting to add inline code tags around code text, let's make this change too if you don't mind.
```suggestion
`happy(32)` should return `true`.
```